tests/naming/src/lib.rs

```diff
@@ -70,7 +70,7 @@ trait T1 {}
 impl T1 for Foo {}
 impl T1 for (Foo, u8) {}
 
-trait T2_for_a {}
-impl T2_for_a for Arity1<(Foo, u8)> {}
-trait T3_e_for_a {}
-impl T3_e_for_a for Foo {}
+trait T2ForA {}
+impl T2ForA for Arity1<(Foo, u8)> {}
+trait T3EForA {}
+impl T3EForA for Foo {}
```

Comment:
Please revert and add `#[warn(non_camel_case_types)]`
This tests the naming translation: while impls have no name in Rust, we need to address them in some backends. An impl `impl Trait for Type` is named `Trait_for_Type` when `Type` is simple enough. Those impls test this naming scheme.

Reply:
fixed